README-dev.md

```diff
@@ -0,0 +1,17 @@
+# Building the docs
+
+The docs build instructions have been tested with Sphinx 4 and Fedora 32.
+
+## Prerequisites
+
+To build and preview the docs locally, you will need to install the following software:
+
+- Git
+- Python 3.7
+- pip
+- Java JDK 6 or above
```

Comment:
I don't think you can successfully build with JDK 6. The code itself is JDK 6 compatible, but due to Maven Central Repository requiring specific TLS standard (unsupported in JDK 6) fetching dependencies will fail.

Reply:
This comes from the README:

> The java driver supports Java JDK versions 6 and above.

I can confirm it is building with JDK 8: https://github.com/scylladb/java-driver/blob/scylla-3.x/.github/workflows/docs-pages%40v2.yaml#L28

```suggestion
- Java JDK 8 or above
```